x-pack/plugins/endpoint/public/applications/endpoint/view/hosts/details/policy_response_friendly_names.ts

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+const responseMap = new Map();
+responseMap.set('success', 'Success');
```

Comment:
the value needs to be `i18n`, so something like:

```suggestion
responseMap.set('success', i18n.translate('some.key.here',  { defaultMessage: 'Success' }));
```